### PR TITLE
chore(website): a few new homepage improvements

### PR DESCRIPTION
**Pull request by TheSisb**
### SlantedBackground component

- https://github.com/twilio-labs/paste/pull/969/commits/a2ed1900b2920157d3bd4bcba17c982843a9b27e I was getting bothered with the gap on the top left that sometimes occurred with the hero when resizing the browser window. I realized that the hero didn't need a slanted background, as the second section's slanted background would give it the illusion of being slanted. It just needed a normal gradient background fill. So I updated our SlantedBackground component to accept another (optional) prop called `skewAngle` so I can set it to `0` for the hero. Problem solved! I also did some tidying, like renaming `angle` to `gradientAngle` to make it more explicit, and passing a second argument to the hook so it is side-effect free / pure.

### Hero animation

- https://github.com/twilio-labs/paste/pull/969/commits/89c8de72096f6f00931f6c1e6e18a5eaa712df80 Tuned the playback speed from 3.5 to 2.4. Feels less rapid now, and more on point.
- https://github.com/twilio-labs/paste/pull/969/commits/3b5b9eaeca0a10662984a022fc4480eadedb6f40 Adjusted size and positioning so it fills more whitespace.

### GetStarted section

- https://github.com/twilio-labs/paste/pull/969/commits/cb7f23c8a97b7cba76c9ec41c922a445e83e9b2f Added a header and better connected the backgrounds between this section and the hero
- https://github.com/twilio-labs/paste/pull/969/commits/d9281118d60c9b2209750df8aef09714606916b5 Animate cards in when in viewport (slide + fade)
- https://github.com/twilio-labs/paste/pull/969/commits/f3bf9f3bf720aabf8c6571a30b5e150a469fbae9 Made Cards equalHeight and added spacing between them on mobile. Updated animation so each card fades in independently on mobile
- https://github.com/twilio-labs/paste/pull/969/commits/997eb4a8f624b8c9f66b1358108afcd0f119e67e Made the card links always bottom align with other card links (in the case of the card with 2 links)

### Popular section

- https://github.com/twilio-labs/paste/pull/969/commits/87afed461c1a1945049b59d62f9294fa1c4f5607 Adjusted backgrounds to mesh better between this section and GetStarted
- https://github.com/twilio-labs/paste/pull/969/commits/5431ac6df58044f24678fbbded19eceb055a5f5d Move files into the homepage folder and animate the cards
- https://github.com/twilio-labs/paste/pull/969/commits/f3bf9f3bf720aabf8c6571a30b5e150a469fbae9 Updated animation so each card fades in independently on mobile
- https://github.com/twilio-labs/paste/pull/969/commits/26fb25243c382abf127a3ac6689575d9d089c349 Added dope doodle animations

### Experiment section

- https://github.com/twilio-labs/paste/pull/969/commits/26fb25243c382abf127a3ac6689575d9d089c349 Added dope doodle animations

### Card component

- https://github.com/twilio-labs/paste/pull/969/commits/87eddf077bac4625488b4deda6a2e87d1245e750 Forwarding ref so we can animate this component

### Refactor

- https://github.com/twilio-labs/paste/pull/969/commits/af481b83659de7cc8785554b74e194cc19be418e Refactored how the VisibilitySensor added in previous commits works to be better.
- https://github.com/twilio-labs/paste/pull/969/commits/83b61f2257958e23206fa0d67e99932a263b1f27 Ran all the images in the website through `imageOptim`, a lossless image compressor. Saved some MBs!

### CODE REVIEW

File: packages/paste-core/components/card/src/index.tsx [hunk 1]
```diff
@@ -4,34 +4,29 @@ import {PaddingProps, isPaddingTokenProp} from '@twilio-paste/style-props';
...
-      paddingLeft={paddingLeft}
-      paddingRight={paddingRight}
-      paddingTop={paddingTop}
-      backgroundColor="colorBackgroundBody"
-    >
-      {children}
-    </Box>
-  );
-};
+const Card = React.forwardRef<HTMLElement, CardProps>(
```
Comment (SiTaggart) [inline packages/paste-core/components/card/src/index.tsx:7]: fast coming to the conclusion we should basically make this a requirement for all components to forward refs

Comment (TheSisb) [reply packages/paste-core/components/card/src/index.tsx:7]: Yeah. I'll make a ticket to do a forwardRef pass. It's pretty important

**Review by SiTaggart — COMMENTED** [review]
I can't tell if I'm reading this correctly, but it looks like you deleted a bunch of images?

**Review by SiTaggart — APPROVED** [review]
Silky

File: packages/paste-website/src/components/SlantedBackgroundGradient.tsx [hunk 2]
```diff
@@ -3,51 +3,60 @@ import {BackgroundColorOptions} from '@twilio-paste/style-props';
...
   ...props
 }) => {
-  const [skewOffset] = useSlantedSkew();
+  const [skewOffset] = useSlantedSkew(skewAngle);
   const {backgroundColors} = useTheme();
 
   return (
     <div
-      {...props}
+      aria-hidden
```
Comment (SiTaggart) [inline packages/paste-website/src/components/SlantedBackgroundGradient.tsx:39]: I think this is hiding the entire hero section from screen readers. I can't get to it at all on this branch

**Pull request merged**